test/WebJobs.Script.Tests/Artifacts/ApiHub/ApiHubTestHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.ApiHub.Sdk;
+using Microsoft.Azure.ApiHub.Sdk.Table;
+using Microsoft.Azure.ApiHub.Sdk.Table.Internal;
+using Microsoft.Azure.WebJobs.Extensions.ApiHub.Common;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Artifacts.ApiHub
+{
+    public static class ApiHubTestHelper
+    {
+        public const string Key = "AzureWebJobsSql";
+        public const string DataSetName = "default";
+        public const string TableName = "SampleTable";
```

Comment:
The tests I have can run with both real environment and mocks, depending on whether the AzureWebJobsSql environment variable is set to a connection string or not. When run against a real environment they assume the SampleTable exists with the correct schema (I put a comment about that). Unfortunately the ApiHub tabular connectors do not have APIs to create/delete tables, just rows, which limits the things I can do. The test setup creates a row if it doesn't exist and the functions update that row. In addition they are run as part of the same method, not concurrently. Generally I agree with you but at the moment I am bound by these constraints.